app/models.py

```diff
@@ -652,7 +652,6 @@ def __init__(self, **kwargs):
 
     __table_args__ = (
         UniqueConstraint('service_id', 'callback_type', name='uix_service_callback_type'),
-        UniqueConstraint('service_id', 'callback_channel', name='uix_service_callback_channel'),
```

Comment:
How is this no longer necessary?

Reply:
I was blindly changing the code to get rid of the unexpected outputs. Now I feel like this is necessary, but I am still wondering why some changes has not been applied to the database or some unexpected change already applied to the database without the migration script.